core/controllers/topic_editor_test.py

```diff
@@ -478,15 +478,11 @@ def test_editable_topic_handler_put_fails_with_long_commit_message(self):
         self.login(self.CURRICULUM_ADMIN_EMAIL)
         csrf_token = self.get_new_csrf_token()
 
-        json_response = self.put_json(
+        self.put_json(
             '%s/%s' % (
                 feconf.TOPIC_EDITOR_DATA_URL_PREFIX, self.topic_id),
             change_cmd, csrf_token=csrf_token, expected_status_int=400)
 
-        self.assertEqual(
-            json_response['error'],
-            'Commit messages must be at most 375 characters long.')
-
```

Comment:
Why is this removed?

Reply:
Ideally I shouldn't be modifying these tests for what I changed in `topic_editor.py`. I was thinking we should not tie our tests to the actual error message but a test to verify 400 error solves the purpose, so in future if schema_utils changes the validation message, we don't need to modify `topic_editor_tests`, feels like an unwanted dependency. 
 
If that is a bigger decision to make, for now I can change the assertion to match the new validation message like the one below. Please let me know.

```
Schema validation for 'commit_message' failed: Validation failed: has_length_at_most ({'max_value': 375}) for object aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Reply:
I think it is fine to verify the actual message, as I don't think this part will change that much an we can always change the tests.

Reply:
Thanks added the test back.